### PR TITLE
Form functionality

### LINKED ISSUE #27: Implement functionality of submission form

### PR DESCRIPTION
**Pull request by Karyum**
close #27 
### Sorry for the big pull request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ config.env
 node_modules/
 npm-debug.log
 coverage/
+webpack.config.js
```
Comment (Suuuuha) [inline .gitignore]: Why you moved it here??

Comment (Karyum) [inline .gitignore]: i guess we dont need it anymore ill just delete it

File: database/schema.js [hunk 2]
```diff
@@ -37,6 +36,5 @@ module.exports = {
   eCommerce,
   education,
   cities,
-  other,
```
Comment (Suuuuha) [inline database/schema.js]: why did you remove that??

File: src/views/startup_submission.hbs [hunk 3]
```diff
@@ -15,8 +15,8 @@
   </fieldset>
 
   <fieldset>
-    <label for="description_textarea">Startup Description:</label>
-    <textarea rows="8" cols="80" id="description_textarea" required></textarea>
+    <label for="description">Startup Description:</label>
```
Comment (Suuuuha) [inline src/views/startup_submission.hbs]: Could you please remove all ":" ??

File: src/views/startup_submission.hbs [hunk 4]
```diff
@@ -15,8 +15,8 @@
...
+    <input type="text" name="description" id="description">
```
Comment (Suuuuha) [inline src/views/startup_submission.hbs]: I think we should still keep description as required

File: src/routes/register_startup.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    startupName: req.body['startup-name'],
+    founderName: req.body['founder-name'],
+    startupDescription: req.body['description'],
+    startupField: req.body.Select,
+    startupStage: req.body['startup-stage'],
+    founderEmail: req.body['startup-email'],
+    startupWebsite: req.body['startup-website']
+  });
+
+  console.log(startup);
```
Comment (esraajb) [inline src/routes/register_startup.js]: lets delet this

File: src/views/startup_submission.hbs [hunk 6]
```diff
@@ -5,22 +5,22 @@
...
 
   <fieldset>
-    <label for="startup-name">Startup Name: </label>
+    <label for="startup-name">Startup Name </label>
     <input type="text" name="startup-name" id="startup-name" required>
   </fieldset>
 
   <fieldset>
-    <label for="founder-name">Founder/Co-Founder name: </label>
+    <label for="founder-name">Founder/Co-Founder name </label>
```
Comment (esraajb) [inline src/views/startup_submission.hbs]: let's remove co-founder in the meantime.
we should have some functionality here, but we will tackle it later.

File: src/views/startup_submission.hbs [hunk 7]
```diff
@@ -5,22 +5,22 @@
...
   <fieldset>
-    <label for="description_textarea">Startup Description:</label>
-    <textarea rows="8" cols="80" id="description_textarea" required></textarea>
+    <label for="description">Startup Description</label>
+    <input type="text" name="description" id="description" required>
   </fieldset>
 
   <fieldset>
-    <label for="select-field">Startup Field: </label>
+    <label for="select-field">Startup Field </label>
```
Comment (esraajb) [inline src/views/startup_submission.hbs]: umm so we are working with the word 'sector' now, so let's be consistent.

File: src/views/startup_submission.hbs [hunk 8]
```diff
@@ -5,22 +5,22 @@
 <form class="" action="/registerStartup" method="post">
 
   <fieldset>
-    <label for="startup-name">Startup Name: </label>
+    <label for="startup-name">Startup Name </label>
```
Comment (esraajb) [inline src/views/startup_submission.hbs]: Im not sure that we need the word Startup before each property.
what do you think about just leaving it as Name, Description, Stage..?

File: tests/index.js [hunk 9]
```diff
@@ -1,5 +1,7 @@
 var test = require('tape');
 
+const serverSide = require('./server/index.js');
+
 test('should return -1 when the value is not present in Array', function (t) {
```
Comment (esraajb) [inline tests/index.js]: do we need this test now ?

File: src/views/startup_submission.hbs (hunk 8 above)
Comment (Karyum) [inline src/views/startup_submission.hbs]: yeah sounds good

File: package.json [hunk 10]
```diff
@@ -24,13 +24,17 @@
   "devDependencies": {
     "codecov": "^2.2.0",
     "istanbul": "^0.4.5",
+    "supertest": "^3.0.0",
     "tape": "^4.6.3"
   },
   "dependencies": {
+    "bluebird": "^3.5.0",
```
Comment (esraajb) [inline package.json]: where do you use bluebird?

Comment (Karyum) [inline package.json]: @esraajb so i added bluebird just cause everytime i save something into the database it logs that i need to use my own promise library so i thought that's why my form didn't work. but now its works and i can remove it but it will still send that error that i need to use my own library is it fine ?

File: tests/index.js (hunk 9 above)
Comment (Karyum) [inline tests/index.js]: i don't think so ill  just comment it out

Comment (Karyum) [inline tests/index.js]: @esraajb did you mean the serverSide tests ? or the basic test in the main index.js?
